pynicotine/config.py

```diff
@@ -315,10 +315,10 @@ def load_config(self):
                 "filtercc": [],
                 "filterin": [],
                 "filterout": [],
-                "filtersize": [],
-                "filterbr": [],
-                "filtertype": [],
-                "filterlength": [],
+                "filtersize": [">50MiB", ">20MiB|<50MiB", ">10MiB|<20MiB", ">5MiB|<10MiB", "<=5MiB"],
+                "filterbr": ["!0", "128|<224", ">192|<=320", "=320|=1411", ">320"],
+                "filtertype": ["flac|wav|aiff|cue|iso", "mp3|m4a|aac|ogg|opus", "mp4|m4v|mkv|mov|264", "!mp3|!wma]"],
```

Comment:
Could you include webm in the video filters?

Reply:
The string results in a list entry that is too long to fit in the combobox dropdown.

Reply:
The issue will be resolved by https://github.com/nicotine-plus/nicotine-plus/pull/2383